openedx_caliper_tracking/tasks.py

```diff
@@ -115,16 +170,20 @@ def send_system_recovery_email(self):
     """
     Send system recovery report to specified email address.
     """
+    reporting_email = settings.CALIPER_KAFKA_SETTINGS.get('ERROR_REPORT_EMAIL')
```

Comment:
Is it an email or an email address?

Reply:
What's the difference? 
This is the email address I guess like abc@def.com

Reply:
reporting_email_address may be a more appropriate name then??

Reply:
Don't think it makes a difference.